Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -44,21 +50,24 @@ common library. It opens the ways to re-use the HTML rendering
 code between our JVM server-side and JS client-side.
 
 We may need to refresh the Gradle project model now to make sure
```

Comment:
We should refresh the Gradle project model now to make sure
the IDE sees all our project model changes. Let's click on the _refresh_
icon in the Gradle tab to accomplish this.